pipeline/outputs/influxdb.md

```diff
@@ -191,6 +192,43 @@ pipeline:
 {% endtab %}
 {% endtabs %}
 
+### Prefix stripping
+
+When collecting data from many inputs into many buckets, it can be helpful to remove a common prefix using `Strip_prefix`.
+
+```python
+[INPUT]
+    Name  cpu
+    Tag   cpu.one
+
+[INPUT]
+    Name  cpu
+    Tag   cpu.two
+
+[INPUT]
+    Name  cpu
+    Tag   gpu.one
+
+[INPUT]
+    Name  cpu
+    Tag   gpu.two
+
+[OUTPUT]
+    Name          influxdb
+    Match         cpu*
+    Bucket        cpubucket
+    Strip_prefix  cpu.
+
+[OUTPUT]
+    Name          influxdb
+    Match         gpu*
+    Bucket        gpubucket
+    Strip_prefix  gpu.
+```
+
+This will result in the buckets `cpubucket` and `gpubucket` each containing two measurement streams named `one` and `two`. Without prefix stripping, the measurement names would be `cpu.one` and `cpu.two (stored in `cpubucket`), and `gpu.one`, `gpu.two` (stored in `gpubucket`).
```

Comment:
```suggestion
This will result in the buckets `cpubucket` and `gpubucket` each containing two measurement streams named `one` and `two`. Without prefix stripping, the measurement names would be `cpu.one` and `cpu.two` (stored in `cpubucket`), and `gpu.one`, `gpu.two` (stored in `gpubucket`).
```

miscounted my backticks.